osprey/config_templates.py

```diff
@@ -0,0 +1,105 @@
+from collections import OrderedDict
+
+params_config_template = OrderedDict(
```

Comment:
Thanks for moving these in here. Is there a reason why you decided to go with `OrderedDict`?

Reply:
Functionally, there isn't really any difference in returning `dictionary` or `OrderedDict`. However, the `read_config` function of RVIC returns `OrderedDict` and I wanted to make sure `config_handler` is returning the same type.